app/uk/gov/hmrc/cdsreimbursementclaimfrontend/views/NotSubscribedToCds.scala.html

```diff
@@ -17,43 +17,36 @@
 @import uk.gov.hmrc.cdsreimbursementclaimfrontend.config.AppConfig
 @import play.api.i18n.Messages
 
-@this(layout: Layout)
+@this(
+        layout: Layout,
+
+)
 
 @()(implicit messages: Messages, appConfig: AppConfig)
 
 @layout(pageTitle = Some(messages("cf.not-subscribed-to-cds.detail.title"))) {
+
     <div id="content" class="column-two-thirds">
         <header>
             <h1 class="govuk-heading-xl">@messages("cf.not-subscribed-to-cds.detail.heading")</h1>
         </header>
-        <p class="govuk-body">
-        @messages("cf.not-subscribed-to-cds.detail.info")
-        </p>
+
+        <div class="govuk-inset-text">
+            <p class="govuk-body">
+                 @messages("cf.not-subscribed-to-cds.detail.info")
+            </p>
+        </div>
+
+        <h3 class="govuk-heading-s">@messages("cf.not-subscribed-to-cds.detail.eori-info-heading")</h3>
 
         <p class="govuk-body">
-            @messages("cf.not-subscribed-to-cds.detail.eori-info")
+            @messages("cf.not-subscribed-to-cds.detail.eori-info-p1")
             <a class="govuk-link" href=@(appConfig.registerCdsUrl)>
-            @messages("cf.not-subscribed-to-cds.detail.eori-link")
+                @messages("cf.not-subscribed-to-cds.detail.eori-link")
             </a>
-            .
+            @messages("cf.not-subscribed-to-cds.detail.eori-info-p2")
```

Comment:
It's generally a good idea to make a message key here which you can reference repeatedly.